terraform/modules/emr/iam.tf

```diff
@@ -573,3 +590,71 @@ data "aws_iam_policy_document" "elastic_map_reduce_for_auto_scaling_role" {
     resources = ["*"] // Required by AutoScaling-CheckPermissions 
   }
 }
+
+data "aws_iam_policy_document" "group_hive_data_access_documents" {
+  for_each = var.security_configuration_groups
+
+  policy_id = "HiveData${replace(regex("((Non_)?PII)", each.key)[0], "_", "")}"
```

Comment:
Whats this Non_ PII references?  Why are we hardcoding parts of group names? There will be new groups added in future, what does that mean for this regex?

Reply:
Sorry - I meant to amend the regex to be similar to [this sed](https://github.com/dwp/aws-analytical-env/pull/193/files/5883cc6606e6513180f053e2c00d6fe4471ed3e8#diff-301f04c512e663a7e7389eea0d85f032facb55a7ad5e812f86cefcccceb3713aR42) to catch any non-alphanumeric chars but missed it when I raised the PR. Will sort now.